src/actions.js

```diff
@@ -234,12 +236,15 @@ export function graphqlMutation(mutation, variables, type = "CORE_TRIGGER_MUTATI
 }
 
 export function fetch(config) {
+  const csrfToken = localStorage.getItem('csrfToken');
   return async (dispatch) => {
     return dispatch({
       [RSAA]: {
         ...config,
         headers: {
           "Content-Type": "application/json",
+          'X-CSRFToken': csrfToken,
+          'X-Requested-With': REQUESTED_WITH,
```

Comment:
Here addition of 'X-REQUESTED-WITH'. It's used in backend to check source and distinguish csrf check on backend level to not block requests coming from android.  @delcroip